### PR TITLE
Replace raw with psbt transactions to support parital tx signing

### PR DESCRIPTION
**Pull request by obucina**

### CODE REVIEW

File: libraries/chain/include/graphene/chain/protocol/sidechain_transaction.hpp [hunk 1]
```diff
@@ -1,60 +1,72 @@
...
+        (son_wallet_id)
+        (son_wallet_deposit_id)
+        (son_wallet_withdraw_id)
+        (transaction)
+        (signers) )
+
+FC_REFLECT( graphene::chain::sidechain_transaction_sign_operation::fee_parameters_type, (fee) )
+FC_REFLECT( graphene::chain::sidechain_transaction_sign_operation, (fee)(payer)
+        (sidechain_transaction_id)
+        (transaction) )
```
Comment (gladcow) [inline libraries/chain/include/graphene/chain/protocol/sidechain_transaction.hpp]: Shoud we serialize `complete` field too?

**Comment by gladcow** [comment]
`get_active_son_wallet` doesn't returns bitcoin address info for wallet,  sons list only

**Comment by obucina** [comment]
> > `get_active_son_wallet` doesn't returns bitcoin address info for wallet,  sons list only
> 
> Sorry, it was my fault, it is working

FYI, this is how it works - The wallet is recreated during maintenance period, if conditions for wallet recreation are met. Then, its on a sidechain handlers to update the son_wallet_object with the addresses of shared wallet on a sidechain they are handling. Usually, updating son_wallet_object happens in matter of seconds - first son that notice empty object and is able to update it, will update it. If get_active_son_wallet does not show sidechain addresses, the son_wallet_object is not yet updated.

File: libraries/chain/include/graphene/chain/protocol/sidechain_transaction.hpp (hunk 1 above)
Comment (obucina) [reply libraries/chain/include/graphene/chain/protocol/sidechain_transaction.hpp]: Done

File: libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp [hunk 2]
```diff
@@ -664,109 +867,80 @@ void sidechain_net_handler_bitcoin::recreate_primary_wallet() {
...
+
+               std::string tx_str = create_transaction(inputs, outputs);
+
+               if (!tx_str.empty()) {
+
+                  auto active_sons = gpo.active_sons;
+                  std::vector<son_id_type> signers;
+                  for (const son_info &si : active_sons) {
+                     signers.push_back(si.son_id);
+                  }
```
Comment (gladcow) [inline libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp]: We should use a set of active sons for previous wallet,  not for the new one to sign transfer transaction (it is transaction from the old wallet to the new wallet,  so the old wallet owners should sign it). What is stored in the `gpo.active_sons` at this moment?

Comment (obucina) [reply libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp]: Good catch. I'll fix it asap.

**Pull request merged**